### PR TITLE
main section draft

### PR DESCRIPTION
**Pull request by IevaZR**

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -1,4 +1,66 @@
 * {
-    margin: none;
-    padding: none
+    margin: 0;
```
Comment (piole007) [inline stylesheet.css:2]: I am wondering if this doesn't duplicates the code? ¨margin: none¨ and ¨margin: 0¨? The same with padding.

File: index.html [hunk 2]
```diff
@@ -1,18 +1,40 @@
...
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Fashion Lookbook</title>
-    <link rel="stylesheet" href="stylesheet.css">
-</head>
-<body>
+    <link rel="stylesheet" href="stylesheet.css" />
```
Comment (piole007) [inline index.html:7]: I think the links to stylesheets should go inside <head> element.

**Review by piole007 — COMMENTED** [review]
Hey! Just added two comments regarding stylesheet link in index.html and margin7padding properties in stylesheet. Can you take a look at those two, please?

File: stylesheet.css (hunk 1 above)
Comment (IevaZR) [reply stylesheet.css:2]: I think they are not duplicating. The red shows what has been removed and the green ones are added. So in this case I corrected the mistake - removed margin: none and padding: none and replaced them with margin: 0 and padding: 0

File: index.html (hunk 2 above)
Comment (IevaZR) [reply index.html:7]: The same as with the above lines - lines 7-9 have been removed and the green ones have been added

**Comment by IevaZR** [comment]
Hello! Added replays to your comments :)

File: stylesheet.css (hunk 1 above)
Comment (piole007) [reply stylesheet.css:2]: Great! I didn't get it about red/green colors 😄

**Pull request merged**